gammapy/scripts/sherpa_like.py

```diff
@@ -93,11 +93,17 @@ def sherpa_image_like(counts,
     # ---------------------------------------------------------
     # Fit and save information we care about
     # ---------------------------------------------------------
-
-    # show_all() # Prints info about data and model
+    #sherpa.astro.ui.show_all() # Prints info about data and model
     sherpa.astro.ui.fit()  # Does the fit
-    sherpa.astro.ui.covar()  # Computes symmetric errors (fast)
+    #sherpa.astro.ui.covar()  # Computes symmetric errors (fast)
     # conf() # Computes asymmetric errors (slow)
     # image_fit() # Shows data, model, residuals in ds9
     log.info('Writing {}'.format(outfile))
     write_all(outfile)
+
+    # Save model image
+    sherpa.astro.ui.set_par('background.ampl', 0)
+    sherpa.astro.ui.notice2d()
+    logging.info('Writing model.fits')
```

Comment:
Please follow [this guideline](https://gammapy.readthedocs.org/en/latest/development/howto.html#generating-log-messages) to generate log messages.